bundles/org.openhab.core.automation/src/main/resources/OH-INF/automation/moduletypes/GenericConditions.json

```diff
@@ -66,7 +66,7 @@
 					"type": "TEXT",
 					"description": "the compare operator, allowed are <,>,=,!=,>=,<= matches",
 					"required": true,
```

Comment:
I wonder why we have required set to true here, if we have a default value defined. According to our usual logic, it should be false - same for other occurrences here in this PR (I know, it was like that before, but I am just seeing it now).